Sources/Paywalls/Components/PaywallComponentPropertyTypes.swift

```diff
@@ -11,11 +11,14 @@
 //
 //  Created by James Borthwick on 2024-08-29.
 
-// swiftlint:disable missing_docs todo
+// swiftlint:disable missing_docs
 import Foundation
 
 #if PAYWALL_COMPONENTS
 
+public typealias TierId = String
+public typealias ColorHex = String
```

Comment:
Same here with public typealiases... worried about people getting these since not namespaced

Reply:
Discussion in [above comment](https://github.com/RevenueCat/purchases-ios/pull/4237#discussion_r1744112744), resolving this one.